db/Cargo.toml

```diff
@@ -6,15 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-sqlx = { workspace = true, features = [
-  "postgres",
-  "runtime-tokio-rustls",
-  "chrono",
-  "migrate",
-  "macros",
-  "uuid",
-] }
-color-eyre = { workspace = true }
+sqlx = { workspace = true, features = ["postgres", "runtime-tokio-rustls", "chrono", "migrate", "macros", "uuid"] }
+color-eyre = "0.6.3"
```

Comment:
Lets use the `color-eyre = { workspace = true }` version!

That way we only have a single version of `color-eyre` in the workspace!